achievements/achievements.ts

```diff
@@ -2571,6 +2706,15 @@ const achievements: Achievement[] = [
 		counter: 'kirafan-easy-answer-first-hint',
 		value: 10,
 	},
+	{
+		id: 'kirafan-easy-answer-first-hint-100',
+		difficulty: 'professional',
+		title: '七賢者',
```

Comment:
```suggestion
		title: 'スクライブギルド長',
```
きらファン当てクイズeasyのヒント1回はきららファンタジアオリジナルキャラクターであるところのメディアさんを題材にしているので、実績名はこちらのほうが嬉しい